client/state/signup/steps/design-type/reducer.js

```diff
@@ -11,7 +11,7 @@ import { designTypeSchema } from './schema';
 
 export default createReducer( '',
 	{
-		[ SIGNUP_STEPS_DESIGN_TYPE_SET ]: ( state = '', action ) => {
```

Comment:
Note that i needed to remove this initialization for the `no-unused-vars` not to fail.